src/cdk-constructs/IxBucket.ts

```diff
@@ -12,34 +12,20 @@ export class IxBucket extends Bucket {
     id: ConstructId,
     props: ConstructProps = {},
   ) {
-    let bucketProps: ConstructProps = {}
+    const bucketProps: ConstructProps = {
+      blockPublicACLs: true,
+      cdk: {
+        bucket: {
+          enforceSSL: true,
+          ...props.cdk?.bucket,
+        },
+        ...props.cdk,
```

Comment:
It's confusing but in this case `...props.cdk` should go before `bucket` otherwise if someone uses the props `{cdk: {bucket: {someKey: 'value'}}}` then the default `enforceSSL: true` won't be set. We want people to have the option to override `enforceSSL` but the user should have to be explicit about overriding it (i.e. they should have to include `enforceSSL: false`.